### PR TITLE
implementation of spec

### PR DESCRIPTION
**Pull request by mordechaigellis**
#1 
this is my first try at implementing the spec as sql script

### CODE REVIEW

File: 0 database table sample data.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+create database germs
+go
+create table germ(id int not null identity primary key, germname varchar(10) not null)
```
[the remark this answers is not in the document]
Comment (charliecpu) [reply 0 database table sample data.sql:3]: germname varchar max is too low, germs can have names up to 255 chars

[the remark this answers is not in the document]
Comment (charliecpu) [reply 0 database table sample data.sql:3]: Also you did not supply sample data

[the remark this answers is not in the document]
Comment (mordechaigellis) [reply 0 database table sample data.sql]: Please see my latest commit, I addressed all the issues you raised

[the remark this answers is not in the document]
Comment (charliecpu) [reply 0 database table sample data.sql]: Really nice work. But, you still didn't do the sample data

[the remark this answers is not in the document]
Comment (mordechaigellis) [reply 0 database table sample data.sql]: see my latest commit, sample data added

**Comment by mordechaigellis** [comment]
Mark for Mordechai Gellis is 100%

**Pull request merged**